### PR TITLE
Decrease waiting/hiding time

### LINKED ISSUE #44: Unmount the datatip right after the focus is removed
When I move my mouse cursor out of the variable, the datatip is unmounted after a little bit of delay.
From my experience of using `atom-ide-ui` package, I felt it much better when the datatip was removed right after the mouse got out of the variable.
It would be really nice if `atom-ide-datatip` also implement this behavior.

### LINKED ISSUE #58: Slight delay for showing datatip
There is a slight delay for showing datatips that should be reduced

### PR DESCRIPTION
**Pull request by aminya**
Fixes https://github.com/atom-ide-community/atom-ide-datatip/issues/58, 
Fixes https://github.com/atom-ide-community/atom-ide-datatip/issues/44

This makes showing time faster and snappier.

### CODE REVIEW

**Comment by remagpie** [comment]
This patch seems to solve the issue by reducing the debounce period and executes the callback more frequently.
Would it be ok for the performance?

BTW, 👍 for your work!

**Comment by aminya** [comment]
Yes it feels very good
time

**Comment by aminya** [comment]
We can make a config setting for it if you want it to be tunable.

**Comment by Belar** [comment]
Thanks for the PR.

> Would it be ok for the performance?

> We can make a config setting

Config option sounds good @aminya, let's make it a setting (in a non-breaking fashion, with the original delay as default). This way everyone can make their own decision about delay (even remove it, 0ms) and tweak in case of degraded performance.

By the way, without commit message in a particular format our deployment setup won't pick it up correctly, could you update the commit according to [contributing guidelines](https://github.com/atom-ide-community/atom-ide-datatip/blob/master/CONTRIBUTING.md#commit-message-guidelines)?

**Comment by aminya** [comment]
This is done.

**Comment by aminya** [comment]
@Belar  This is done.

File: package.json [hunk 1]
```diff
@@ -55,16 +55,22 @@
...
-      "title": "Flag denotes if the data tip should be shown automatically when the mouse moves",
-      "description": "If set to true, the data tip is shown as soon as you move your mouse around.",
+      "title": "Show datatip automatically on \"mouse\" hover",
+      "description": "If set to true, the data tip is shown as soon as mouse hovers on a word.",
       "type": "boolean",
       "default": false
+    },
+    "hoverTime": {
+      "title": "Hover Time",
+      "description": "The time that the mouse should hover to show a datatip. Also specifies the time that the datatip is still shown when the mouse moves [ms].",
```
Comment (Belar) [inline package.json]: > that the mouse should [...] the mouse moves

Your fix also works for cursor, right? Could we reflect that in the description?

**Review by Belar — COMMENTED** [review]
Nice, will be great to have this setting

File: package.json (hunk 1 above)
Comment (aminya) [reply package.json]: Yes. I will add it.

Comment (aminya) [reply package.json]: Done https://github.com/atom-ide-community/atom-ide-datatip/pull/61/commits/07a81641d3b7de526923cda7dc4c22e2d1d18c53

**Pull request merged**